### PR TITLE
implement search api

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12856

*Description of changes:*
implemented the business look up API for MHR registration step 3 (Home Owners)
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/public/config/configuration.json [hunk 1]
```diff
@@ -24,5 +24,8 @@
   "ADDRESS_COMPLETE_KEY": "",
   "SYSTEM_MESSAGE": "<a style='text-decoration: underline; color:inherit;' href='https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization/personal-property-registry-transition' target='_blank'>See known issues and updates for Personal Property Registry here</a>.",
   "SYSTEM_MESSAGE_TYPE": "warning",
-  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
+  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
+  "REGISTRIES_SEARCH_API_URL": "https://bcregistry-dev.apigee.net/registry-search",
+  "REGISTRIES_SEARCH_API_VERSION": "/api/v1",
+  "REGISTRY_SEARCH_API_KEY": "oPiLjOE4rvmw556dTextKI7mHqfSd7AF"
```
Comment (cameron-eyds) [inline ppr-ui/public/config/configuration.json:30]: Will need to request that these configurations are added to our 1password i believe. 
Might need to discuss with Doug or Thor or whomever manages Assets configurations.

**Review by cameron-eyds — APPROVED** [review]
LGTM. 
Does it work?

Also please update the PR with a description :)

File: ppr-ui/public/config/configuration.json (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/public/config/configuration.json:30]: Also this an item that should be called out in a future release ticket as a release dependency for each environment.

**Comment by garyfan1** [comment]
I believe so, I've added a screenshot to the PR description.

File: ppr-ui/public/config/configuration.json (hunk 1 above)
Comment (garyfan1) [reply ppr-ui/public/config/configuration.json:30]: ok I'll get in touch with Doug.

**Review by ChasBoucher — APPROVED** [review]
Nice.. Looks good to me..

**Review by doug-lovett — APPROVED** [review]
Looks good, does the account id work with staff/IDIR users?

**Comment by garyfan1** [comment]
the screenshot was taken logged in as myself (IDIR user). The `CURRENT_ACCOUNT` wasn't set so we can't get the account id but the call to API still works.

Thanks.

**Review by dimak1 — APPROVED** [review]
Nice work!

**Pull request merged**